src/api/uploads/PlainUpload.js

```diff
@@ -77,6 +77,7 @@ class PlainUpload extends BaseUpload {
         const attributes = JSON.stringify({
             name: this.fileName,
             parent: { id: this.folderId },
+            description: this.fileDescription,
```

Comment:
Verify that this is not a breaking change and not dependent on a server api change?

Reply:
@j3tan I verified in EUA, where this param is undefined, our upload does not breaking with these changes. Is there anywhere else I should be testing?

Reply:
sounds good, just wanted to make sure this was backwards compatible